docs/psf_spec/high_level_overview.rst

```diff
@@ -0,0 +1,244 @@
+        gs [label = "guess at stars", shape = box];
+        p [label = "psf model"];
+        sm [label = "scene maker"];
+        st [label = "stars\n(astropy.table)", shape = box];
+
+        label = <<FONT POINT-SIZE = "20"><I>photutils.psf </I>class structures</FONT>>;
+        labelloc = top;
+
+        w -> i [style = dashed];
+        // lhead sets the arrow end at the subgraph, rather than at finder itself
+        i -> f [lhead = cluster0];
+        gs -> invis [style = dashed];
+
+        f -> g;
+        f -> invis;
+        invis -> g;
+        som -> f [style = dashed];
+        n -> ft;
+        som -> ft;
+        g -> ft;
+        b -> f [style = dashed];
+        b -> g [style = dashed];
+        b -> ft [style = dashed];
+        ft -> c [style = dashed, label = "Subtract", dir = none];
+        c -> f [style = dashed];
+
+        i -> gs [style = invis];
+        
+        p -> som;
+        sm -> g [style = dotted];
+        ft -> st;
+        // reverse arrow to put psf model above scene maker, saving space with scene
+        // maker next to the middle of the main box
+        p -> sm [style = dotted, dir = back];
+
+        // puts things that should be at the top of the box at the top for orientation
+        // and structuring; requires 'newrank' above to set subgraph items equal
+        {rank = same; w; som;}
+        {rank = same; p; f; n;}
+        {rank = same; gs; g; sm;}
+    }
+
+
+.. _Image:
+
+Input Image
+^^^^^^^^^^^
+
+The input image is the most fundamental part of the PSF fitting process, as it is the
+product on which all subsequent methodology is applied. The image must be a two-dimensional
+array of ``x`` and ``y`` coordinates with ``flux`` represented by array values. If a
+WCS -- contained within a FITS file, for instance -- is passed with the two-dimensional
+array then the position of sources can be given in sky coordinates, instead of pixel values.
+The given image is passed as an input to the chosen PSF fitting class.
+
+
+.. _Finder:
+
+Finder
+^^^^^^
+
+The finder is the first step in the PSF fitting process, as sources must be discovered in
+the image before any kind of fit can be applied to them. These accept the input ``image``
+and produce a `~astropy.table.Table` of detected sources, by a set of criteria internal to
+the given ``Finder`` (see, e.g., `~photutils.detection.DAOStarFinder` using ``roundness``
+and ``sharpness`` to determine if sources are point-like). If an initial set of detected
+sources is passed to the fitter as ``init_guesses`` then ``Finder`` is not run on the first
+pass of an iterative fitting class, instead using the provided positions and fluxes. See
+the :doc:`finder <finder>` documentation for more details.
+
+
+.. _Group Maker:
+
+Group Maker
+^^^^^^^^^^^
+
+The second block to run in the fitting process, the various group maker processes, such as
+`~photutils.psf.DAOGroup`, allow for the merging of astrometrically overlapping sources
```

Comment:
```suggestion
 `~photutils.psf.DAOGroup`, allow for the merging of sources that are astrometrically nearby
```

"overlapping" is a bit unclear because the definition of "overlapping" somewhat depends on the algorithm.  I think "nearby" is better in that it's intentionally vague?